### ISSUE TITLE
feat: report match totals for each test-file based on all reference files

### ISSUE THREAD

**Issue by ankostis** [blingenf/copydetect#44]
Currently  (v0.4.5) the tool reports match-ratios between all  pairs of _test_ <--> _ref_ files - i will focus here on match-ratios for _test_ files but the same applies for _ref_ files, reversed.

Let's assume these are the reported match-ratios for _test_ files:

```mermaid
  graph LR;
      T1--70%-->R1;
      T1--65%-->R2;
      T1--55%-->R3;
      T2--3%-->R1;
      T2--2%-->R2;
      T2--8%-->R3;
```

What  i'm missing is a nec section with all the grand total matchings for each _test_ file vs the whole _ref codebase_,
ie. how many lines are copies, regardless of which specific _ref file_, something like this:

```mermaid
  graph LR;
      T1--98%-->R[R1+R2+r3] ;
      T2--11%-->R[R1+R2+r3] ;
```

When expanding the ratios in these sections I would expect to see only the "left" diff pane with the copied test-code, like a code-coverage report, reporting the number of matches for each LoC, like this:
![image](https://github.com/blingenf/copydetect/assets/501585/744cd7ed-b566-4194-95a0-b540f3fab44e)

Does that make sense?

### Workaround

Currently i have to concatenate all ref-files into a single one with a command like:
```bash
mkdir /tmp/all
find  ref_project/ -name *.cs | xargs cat /tmp/all/all.cs
```
... and then run against the new ref-folder:
```bash
copydetect -t test_prolect/ -r /tmp/all/ -e cs
```

**Comment by blingenf** [comment]
I agree that similarity to the test set as a whole would be useful to display. The improved bookkeeping I have planned for #34 will make this much easier to implement -- I'll address that first then see if I can get something working for aggregate similarity.

**Comment by ankostis** [comment]
Is this related to this comment in [#33](33#issuecomment-1477637551)?

> ...finding the overall matched tokens from all the reference files would be merely impossible?
